File: Ruby/dejavu.rb
str = gets.chomp
stat = ''
i=0
while i <= (str.length - 2)
  if str[i] == str[i + 1]
      stat = 'dejavu'
      break
  else
      stat = 'unique'
  end
  i += 1
end
if stat == 'unique'
  puts 'Unique'
elsif stat == 'dejavu'
  puts 'Deja Vu'
end

File: Python/dejavu.py
text = input()
stat = ''

i=0
while i <= (len(text) - 2):
  if text[i] == text[i + 1]:
      stat = 'Deja Vu'
      break
  else:
      stat = 'Unique'
  i += 1
print(stat)
